communications/README.md

```diff
@@ -0,0 +1,9 @@
+# Communications
+
+## Async written
+
+- If you want to discuss an idea (e.g. new or existing product feature, update a marketing page), comment on or create a new story/epic in Clubhouse. @-mention specific individuals and ask them specific questions. See [Product Development](../product-development/README.md).
+- Ensure your [Clubhouse notifications](https://help.clubhouse.io/hc/en-us/sections/201543416-Notifications) are properly configured. Respond to Tilters within a few days with an answer, or at least say you don't have time currently to do so.
+- If you want to discuss a process change, consider the smallest reasonable change and create a GitHub PR in this repo as a proposal. @-mention and assign specific individuals for review.
+- If you are having difficulty understanding each other over Clubhouse and/or GitHub, consider scheduling a meeting with a specific agenda to further discuss. But prefer starting with async communications first.
+- Slack is an async tool. Don't expect people to respond immediately. Don't feel pressured to respond immediately.
```

Comment:
Do we expect that everyone will even SEE a (non-DM) communication in Slack? might be worth it to set an expectation around that.

Reply:
Thanks @maiamcc. I have more opinions on how Slack should be used. We can definitely collaborate on that in a future PR including what you mentioned. This PR is intended to be a small first step.